spec/services/idv/duplicate_ssn_finder_spec.rb

```diff
@@ -144,4 +147,60 @@
       end
     end
   end
+
+  describe '#associated_facial_match_profiles_with_ssn' do
+    let(:ssn) { '123-45-6789' }
+    let(:user) { create(:user) }
+    let(:user2) { create(:user) }
+    let(:sp) { 'urn:gov:gsa:openidconnect:inactive:sp:test' }
+
+    subject { described_class.new(ssn: ssn, user: user, issuer: sp) }
+
+    before do
+      allow(IdentityConfig.store).to receive(:eligible_one_account_providers)
+        .and_return([
+                      'urn:gov:gsa:openidconnect:inactive:sp:test',
```

Comment:
Can we use different service providers? The "inactive" in the name is misleading.

Reply:
FYI, I am cleaning up the existing problem here - https://github.com/18F/identity-idp/pull/12377.